packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/tera-calibrate-ensemble-ciemss-drilldown.vue

```diff
@@ -459,11 +472,19 @@ const isRunInProgress = computed(() => Boolean(inProgressCalibrationId.value ||
 
 const datasetId = computed(() => props.node.inputs[0].value?.[0] as string | undefined);
 const currentDatasetFileName = ref<string>();
+const dataset = shallowRef<Dataset | null>(null);
+const csvAsset = shallowRef<CsvAsset | undefined>(undefined);
 const datasetColumnNames = computed(
 	() =>
 		dataset.value?.columns?.filter((col) => col.fileName === currentDatasetFileName.value).map((col) => col.name) ??
 		([] as string[])
 );
+const outputDatasetId = computed(() => {
+	if (!selectedOutputId.value) return '';
+	const output = props.node.outputs.find((o) => o.id === selectedOutputId.value);
+	return output?.value?.[0] ?? '';
```

Comment:
Can use null assertion here instead of option chaining, the latter is expensive after transpile, the former just disappears

```
return output!.value?.[0] ?? '';
```

Reply:
Thanks good catch!
I have updated this